PrimerasPracticasJS/ImprimirValorAleatorio.js

```diff
@@ -0,0 +1,9 @@
+/*IMPRIMIR UN VALOR ALEATORIO ENTRE 0 Y 99*/
+let min = 1
+let max = 99
+
+let NumPosibilidades = max - min
+let NumAleatrio = Math.random() * (NumPosibilidades)
+NumAleatrio = Math.floor(NumAleatrio)
```

Comment:
para la declaracion de variables usa la nomenclatura camelCase
``` **Lower Camel Case **–> Está se usa para nombrar nombres de variables y métodos. Para nombrar las variables y métodos comenzamos los nombres con la primera letra en minúsculas, y si el nombre se compone de más de una palabra, las separamos con la primera letra en mayúsculas. ```

Tambien el nombre de las variables siempre trata que sean en ingles, por favor

```suggestion
let numPossibilities = max - min
let numRandom = Math.random() * (numPossibilities)
numRandom = Math.floor(numRandom)
```